src/SourceLink.Git.IntegrationTests/CloudHostedProvidersTests.cs

```diff
@@ -76,12 +76,14 @@ public void NoRepository_NoWarnings()
                 expressions: new[]
                 {
                     "@(SourceRoot)",
+                    // "@(SourceRoot->'%(BranchName)')", // TODO: Unclear what to do here since this test explicitly uses the SDK version of Microsoft.Build.Tasks.Git
```

Comment:
Let's file an issue and link it from the TODO comment. Once the SDK is updated with new Source Link version this should work, right?

Reply:
Done!